### ISSUE TITLE
Using Ein cause Emacs to freeze.

### ISSUE THREAD

**Issue by Vvkmnn** [millejoh/emacs-ipython-notebook#334]
### Summary

Running notebook cells seems to work, but randomly crashes Emacs on idle or during operation: 

![image](https://user-images.githubusercontent.com/1954615/45105933-6f973980-b103-11e8-96c6-4cde7b167105.png)

Luckily Jupyter doesn't crash this time. Though that might have to do with me using Anaconda environments, so I'm taking a look at [this fix](https://emacs.stackexchange.com/questions/32103/spacemacs-use-anaconda-environment) post restart:

![image](https://user-images.githubusercontent.com/1954615/45105953-78880b00-b103-11e8-9583-440b4b8945a7.png)

Looks very similar to https://github.com/millejoh/emacs-ipython-notebook/issues/289, was hoping the fix in https://github.com/millejoh/emacs-ipython-notebook/issues/325 would work, but not on my machine unfortunately.

### System 
* OS: macOS High Sierra
* Emacs: 26.1
* Spacemacs: 0.300.0
* Spacemacs branch: develop (ca8aa4e9b954cc89d6298beaaf2bd3e4d86f04d2)
* Graphic display: t
* Distribution: spacemacs
* Editing style: vim
* Completion: ivy

### Config
```elisp
   dotspacemacs-configuration-layers '(
     ;; Languages
     emacs-lisp
     typescript
     javascript
     clojure
     markdown
     ipython-notebook
     (python :variables
             python-enable-yapf-format-on-save t)
     shell-scripts
     ;; extra-lang
     html
     csv

     ;; Emacs
     ivy
     ;; smex
     ibuffer
     gtags
     vinegar
     evil-commentary
     (shell :variables
            shell-default-shell 'eshell
            shell-default-height 30
            shell-default-position 'bottom)
     (git :variables
          git-magit-status-fullscreen t
          git-enable-github-support t
          git-gutter-use-fringe t)
     colors
     (auto-completion :variables
                      auto-completion-enable-help-tooltip t
                      auto-completion-enable-snippets-in-popup t
                      auto-completion-enable-sort-by-usage t)
     spell-checking
     syntax-checking
     (version-control :variables
                      version-control-diff-tool 'diff-hl
                      version-control-global-margin t)
     (org :variables
          org-enable-github-support t
          org-enable-reveal-js-support t)

     ;; Environment
     dash
     osx
     ;; slack
     ;; xkcd
     ;; emoji
  )
```

### Package

Currently using the September release: 

```elisp  
ein is an installed package.

     Status: Installed in ‘ein-20180904.1447/’ (unsigned). Delete
    Version: 20180904.1447
    Summary: Emacs IPython Notebook
   Requires: websocket-1.7, auto-complete-1.4.0, request-0.3, deferred-0.5, request-deferred-0.2.0, cl-generic-0.3, dash-2.13.0, s-1.11.0, skewer-mode-1.6.2
Other versions: 20180904.1447 (melpa).
```

**Comment by millejoh** [comment]
In the log for the running jupyter server are you seeing errors similar to:

```
[IPKernelApp] ERROR | Exception in message handler:
Traceback (most recent call last):
  File "/home/ubuntu/.conda/envs/jupyter/lib/python3.6/site-packages/ipykernel/kernelbase.py", line 233, in dispatch_shell
    handler(stream, idents, msg)
  File "/home/ubuntu/.conda/envs/jupyter/lib/python3.6/site-packages/ipykernel/kernelbase.py", line 451, in inspect_request
    reply_content = self.do_inspect(content['code'], content['cursor_pos'],
KeyError: 'code'
```

**Comment by millejoh** [comment]
Note that I've pushed a commit that resolves that issue on my setup. Maybe it will help in yours.
